src/Core/Components/Base/FluentInputBase.cs

```diff
@@ -139,7 +143,7 @@ public abstract partial class FluentInputBase<TValue> : FluentComponentBase, IDi
     /// </summary>
     protected internal FieldIdentifier FieldIdentifier { get; set; }
 
-    internal bool FieldBound => Field is not null || ValueExpression != null || ValueChanged.HasDelegate;
+    internal bool FieldBound { get; set; }
```

Comment:
Why did you change this, why didn't you keep `bool FieldBound => Field is not null || ValueExpression != null || ValueChanged.HasDelegate;` ?

I find the property clearer if it retains its verification criteria.


I found this line in `ListComponentBase` class.
```csharp
FieldBound = ValueChanged.HasDelegate || SelectedOptionChanged.HasDelegate || SelectedOptionsChanged.HasDelegate;`
```

It might be preferable to adapt the `bool FieldBound` to a virtual property and override it in `ListComponentBase`.

Reply:
Done with override now